config/draco-config-install.cmake.in

```diff
@@ -92,6 +86,18 @@ set(Draco_PACKAGE_LIST "@Draco_PACKAGE_LIST@" )
 ## The TPLs enabled for this project
 set(Draco_TPL_LIST "@Draco_TPL_LIST@")
 
+## ---------------------------------------------------------------------------
+## Create import targets for TPLs used by Draco (e.g. MPI::MPI_CXX)
+## ---------------------------------------------------------------------------
+
+include(CMakeFindDependencyMacro)
+if ( "${DRACO_C4}" STREQUAL "MPI" )
+  find_dependency( MPI REQUIRED COMPONENTS ${Draco_MPI_LANG_LIST} )
```

Comment:
Let me check on the find_dependency bug to be sure we are protecting ourselves. 

here is the bug: https://gitlab.kitware.com/cmake/cmake/issues/17583

Also, I just read that find_dependency should just do the right thing WRT to keywrods like QUIET or REQUIRED. See discussion here:

https://gitlab.kitware.com/cmake/cmake/issues/18590

Reply:
Have you tested this? Given my experience, it wouldn't surprise me if this fails in super projects that don't enable fortran. This could be avoided by pushing installation support/dependency management down to the level of the components in draco such that a user can find the components of interest.

The easiest solution would be to call enable language where Draco_MPI_LANG_LIST is used, but this would fail for clients that do not have a fortran compiler installed (obviously this is not a typical use case for us but still something to consider).

Reply:
We have two build scenarios where no Fortran compiler is available (Xcode and MSVC) and I have tested this logic in those cases.

You are correct that there is an implicit assumption that the client package will have the same compiler and MPI library dependencies as Draco.  I don't think this assumption is misplaced but maybe you can provide a counter example.

Since _c4_ is a fundamental component in Draco (actually there is no direct dependency on _MPI::MPI_Fortran_ yet but this will be added very soon), I don't see any advantage to pushing the dependency management down. I think this would only serve to increase the complexity of managing import targets.

Let's keep this PR on hold for a bit while I add that bit of code to _c4_.

Reply:
I've commented out this block for now.  I'm still doing some testing in Jayenne.